core/build.gradle

```diff
@@ -120,6 +140,8 @@ dependencies {
     implementation platform("software.amazon.awssdk:bom:${awssdkVersion}")
     implementation 'software.amazon.awssdk:applicationautoscaling'
     implementation 'software.amazon.awssdk:dynamodb'
+    implementation 'software.amazon.awssdk:iam'
```

Comment:
This will add these dependencies to the production Jar and the file size will increase. Why not `testImplementation` or something?